vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -38,37 +12,37 @@
 | Entando Cloud Hub | A SaaS instance of an Entando Hub that contains a public and private collection of components |
 | Entando Component Generator (ECG) | An implementation of a JHipster blueprint that provides automation and templating to generate the components used to build an Entando Application |
 | Entando Component Manager (ECM) | Part of the ECR that coordinates the installation of the MFEs and MSs contained in an Entando Bundle |
```

Comment:
thought about that but this isn't a linear read; it matches our release notes where acronyms can be introduced before they're defined as long as they're defined on their dedicated line

yes, can drop contained... as a general comment, aside from editing out errors and truncating/rephrasing for brevity, i copied over these definitions without making changes; if we're tweaking language here we should in the internal glossary as well

Reply:
FYI, the internal glossary was never tweaked, that needed approval from too many internal people. Only errors were corrected. 

As for the ECR- Is the ECR defines somewhere on the page? This one I do feel strongly about because if people come to a glossary, they don't wan't to find another word or acronyms they have to look up. This is where things should be as clear as possible.

Reply:
fine with me. i'll expand the initialism